jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-pull-json.yaml

```diff
@@ -75,6 +75,11 @@
         job-name: pull-kubernetes-federation-e2e-gce
         repo-name: 'k8s.io/kubernetes'
         timeout: 110
+    - federation-e2e-gce:
+        max-total: 12
+        job-name: pull-federation-e2e-gce
+        repo-name: 'k8s.io/federation'
+        timeout: 110
```

Comment:
@krzyzacy are we allowing new jenkins jobs? :confused:

Reply:
@BenTheElder, this is a temporary measure. we are just moving existing federation jobs to k/f repo. following this pr i am removing existing jobs. I understand that we should move towards adding prow jobs. but its a two step process as of now for federation and also getting bazel working for federation test needs is being worked out. But we cannot wait that longer to move federation code out-of-core.

Reply:
SGTM, I brought it up because we plan to turn down PR-jenkins in particular roughly by 1.9 release (#3778).